canary/apps/angular/angularcli/angular.json

```diff
@@ -19,7 +19,7 @@
             "polyfills": "src/polyfills.ts",
             "tsConfig": "tsconfig.app.json",
             "assets": ["src/favicon.ico", "src/assets"],
-            "styles": ["src/styles.css"],
+            "styles": ["@aws-amplify/ui-angular/theme.css", "src/styles.css"],
```

Comment:
It's weird to me that Angular Authenticator has it's own styling. Do we have a ticket to track removing this?

Reply:
+1

Reply:
I believe this is just the reexport of `@aws-amplify/ui/styles.css`, that this canary app needs to import.

Reply:
I created a ticket